lib/ramble/ramble/workspace/workspace.py

```diff
@@ -859,13 +859,17 @@ def external_spack_env(self, env_name):
     def concretize(self):
         spack_dict = self.get_spack_dict()
 
-        if 'concretized' in spack_dict and spack_dict['concretized']:
-            raise RambleWorkspaceError('Cannot conretize an ' +
-                                       'already concretized ' +
-                                       'workspace')
+        if not self.force_concretize:
+            try:
+                if spack_dict[namespace.packages] or spack_dict[namespace.environments]:
+                    raise RambleWorkspaceError('Cannot concretize an already concretized '
```

Comment:
Can we make a task to track the work of making `concretize` additive as well (which would remove this message)?

Reply:
Added task